File: Lista_Prof_Fabio/Lista_04/q03.js
const input = require('prompt-sync')()

function main(){

    // Entrada
    console.log('--- Calculo MDC ---')
    const num1 = Number(input('--> '))
    const num2 = Number(input('--> '))

    // Processamento
    let numero = 1
    
    while (!(numero % num1 === 0 && numero % num2 === 0)){
        numero++
    }

    // Saída
    console.log(`O MDC entre ${num1} e ${num2} é: ${numero}`)
}

main()

File: 2º_Periodo/Atividade02/q03.c
#include <stdio.h>

int main(){

    int num1,num2,resto,mdc;

    printf("Insira o valor do numero a: ");
    scanf("%i",&num1);
    printf("Insira o valor do numero b: ");
    scanf("%i",&num2);

    int a = num1;
    int b = num2;

    do {
        resto = a % b;

        a = b;
        b = resto;

    } while (resto != 0);

    mdc = a;

    printf("O mdc entre %i e %i eh de %i\n",num1,num2,mdc);

    system("PAUSE");
}